src/Distribution/Server/Features/Browse/Parsers.hs

```diff
@@ -113,6 +113,7 @@ filterWith allowNot = do
         "maintainer:" -> MaintainerFilter <$> wordWoSpaceOrParens
         "deprecated:" -> DeprecatedFilter <$> deprecatedOption
         "distro:" -> DistroFilter <$> wordWoSpaceOrParens
+        "packageRank" -> DownloadsFilter <$> opAndSndParam decimal
```

Comment:
The package rank should have it's own constructor, it makes little sense to use DownloadsFilter

Reply:
For now I undid the changes to Parsers.hs since they do not make sense and are not necessary.